src/main.cpp

```diff
@@ -3125,7 +3126,7 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
     {
         nSigOps += GetLegacySigOpCount(tx);
     }
-    if (nSigOps > MAX_BLOCK_SIGOPS)
+    if (nSigOps > Policy::blockSigOpAcceptLimit(blockSize))
```

Comment:
So this is using the old school sigops count, counting what is in the created outputs, but not what is in the spent outputs ? Is this what BU is doing ?